.github/workflows/push.yml

```diff
@@ -22,14 +22,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        channel: [ "nixos-21.11", "nixos-21.05", "nixos-20.09", "nixos-unstable" ]
+        channel: [ "nixos-21.11", "nixos-21.05", "nixos-20.09", "nixos-unstable"]
```

Comment:
Why the whitespace change here? Is this the correct format?

Reply:
If I did this then it was by accent. Will undo when I squash this branch :)